src/app/products/ecr-viewer/page.tsx

```diff
@@ -0,0 +1,620 @@
+'use client';
+import { ContentContainer } from '@/app/components/ContentContainer/ContentContainer';
+import Hero from '@/app/components/Hero/Hero';
+import { basePath } from '@/app/utils/constants';
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+  Accordion,
+  Link,
+  SideNav,
+} from '@trussworks/react-uswds';
+import classNames from 'classnames';
+import Image from 'next/image';
+import { SetStateAction, useState } from 'react';
+import './styles.scss';
+
+export default function EcrViewer() {
+  return (
+    <div>
+      <Hero
+        header="eCR Viewer"
+        subheader="An intuitive interface that helps epidemiologists and case investigators make better sense of eCR data, faster."
+      />
+      <ContentContainer align>
+        <div className="grid grid-cols-1 gap-6 lg:grid-cols-[1fr_3fr_1fr] lg:gap-0">
```

Comment:
Once we have at least one more product page that is ready to be worked on, I think it'll make sense at that point to break some of these layout elements out into their own components to use across the product pages.